src/validation/fields.ts

```diff
@@ -69,26 +70,58 @@ export const metadata = Joi.object().default({}).example({
   lastName: 'Smith',
 });
 
-export const redirectTo = ENV.AUTH_CLIENT_URL
-  ? Joi.alternatives()
-      .default(ENV.AUTH_CLIENT_URL)
-      .try(
-        ...ENV.AUTH_ACCESS_CONTROL_ALLOWED_REDIRECT_URLS.map((value) =>
-          Joi.string()
-            .lowercase()
-            .regex(new RegExp('^' + value))
-            // * Get the initial value so we don't change the case
-            .raw()
-        ),
-        Joi.string()
-          .lowercase()
-          .regex(new RegExp('^' + ENV.AUTH_CLIENT_URL))
-          // * Get the initial value so we don't change the case
-          .raw()
-      )
-      // ! If validation fails, it will always fall back to `ENV.AUTH_CLIENT_URL` instead of raising an error.
-      .example(`${ENV.AUTH_CLIENT_URL}/catch-redirection`)
-  : Joi.string().uri();
+export const redirectTo = Joi.string()
+  .messages({
+    redirectTo: '{{#label}} is not allowed by the current hasura-auth settings',
```

Comment:
```suggestion
    redirectTo: '{{#label}} is not allowed as a `redirectTo` parameter.',
```

Reply:
`{{#label}}` is always `"redirectTo"`, so it will read:
```
"redirectTo" is not allowed as a `redirectTo` parameter.
```

Reply:
ah ok.